cmd/oras/internal/display/status/track/target.go

```diff
@@ -74,7 +76,7 @@ func (t *graphTarget) Mount(ctx context.Context, desc ocispec.Descriptor, fromRe
 
 // Push pushes the content to the base oras.GraphTarget with tracking.
 func (t *graphTarget) Push(ctx context.Context, expected ocispec.Descriptor, content io.Reader) error {
-	r, err := managedReader(content, expected, t.manager, t.actionPrompt, t.donePrompt)
+	r, err := NewReader(content, expected, t.actionPrompt, t.donePrompt, t.tty)
```

Comment:
A manager controls output to tty and cannot be be shared between `track.Reader`s. I think that's why the unit tests are failing.